bottlecap/src/bin/bottlecap/main.rs

```diff
@@ -804,3 +816,23 @@ async fn setup_telemetry_client(
         .map_err(|e| Error::new(std::io::ErrorKind::InvalidData, e.to_string()))?;
     Ok(telemetry_listener_cancel_token)
 }
+
+fn start_otlp_agent(
+    config: &Arc<Config>,
+    tags_provider: Arc<TagProvider>,
+    trace_processor: Arc<dyn trace_processor::TraceProcessor + Send + Sync>,
+    trace_tx: Sender<SendData>,
+) {
+    if !config.otlp_config_traces_enabled {
+        debug!("OTLP traces agent is disabled, not starting it.");
```

Comment:
Maybe only debug if it's _enabled_? since we'd expect most people to not have it enabled by default